### PR TITLE
Standardize on string quoting

### LINKED ISSUE #941: Cleaning up quote types, i.e. use the same quote type everywhere.
We are using both single quotes and double quotes around text inconsistently. We should use the same quote type everywhere in a module unless there is a specific case not to, for example 'This isn't right' would be better as "This isn't right" even if the rest of the code using single quotes.

### PR DESCRIPTION
**Pull request by mirons-google**
This resolves #941 by using a pylint module and enforcing its use. I've also cleaned up all existing infractions of the now `''` single quote policy.

### CODE REVIEW

File: google/cloud/forseti/common/util/parser.py [hunk 1]
```diff
@@ -59,17 +59,17 @@ def format_timestamp(timestamp_str, datetime_formatter):
         str: The formatted, stringified timestamp.
     """
     try:
-        if '"' in timestamp_str or "'" in timestamp_str:
+        if '\'' in timestamp_str or '\'' in timestamp_str:
```
Comment (joecheuk) [inline google/cloud/forseti/common/util/parser.py]: I think the logic here needs to be updated to `if '"' in timestamp_str or '\'' in timestamp_str` instead

File: google/cloud/forseti/common/util/parser.py [hunk 2]
```diff
@@ -59,17 +59,17 @@ def format_timestamp(timestamp_str, datetime_formatter):
...
             # Make sure the timestamp is not surrounded by any quotes
-            timestamp_str = timestamp_str.replace("'")
-            timestamp_str = timestamp_str.replace('"')
+            timestamp_str = timestamp_str.replace('\'')
```
Comment (joecheuk) [inline google/cloud/forseti/common/util/parser.py]: Same here, one of the statements should be be replacing double quote.

Comment (joecheuk) [reply google/cloud/forseti/common/util/parser.py]: I think this is actually a bug, it should be `timestamp_str.replace('"', '')`

File: google/cloud/forseti/common/util/parser.py [hunk 3]
```diff
@@ -59,17 +59,17 @@ def format_timestamp(timestamp_str, datetime_formatter):
...
+            timestamp_str = timestamp_str.replace('\'')
```
Comment (joecheuk) [inline google/cloud/forseti/common/util/parser.py]: This should be `timestamp_str.replace('\'', '')`

Comment (mirons-google) [reply google/cloud/forseti/common/util/parser.py]: done

File: google/cloud/forseti/common/util/parser.py (hunk 2 above)
Comment (mirons-google) [reply google/cloud/forseti/common/util/parser.py]: done

File: google/cloud/forseti/common/util/parser.py (hunk 1 above)
Comment (mirons-google) [reply google/cloud/forseti/common/util/parser.py]: done

**Pull request merged**